edx_event_bus_kafka/internal/consumer.py

```diff
@@ -462,12 +463,14 @@ def handle(self, *args, **options):
             return
         try:
             signal = OpenEdxPublicSignal.get_signal_by_type(options['signal'][0])
-            if options['offset_time'][0] is not None:
+            if options['offset_time'] and options['offset_time'][0] is not None:
```

Comment:
I thought you already had this? Just curious.

Reply:
This is updating this management command to match the course-discovery version. It was in the course-discovery version and I backported it.